app/bt/nativeModule.ts

```diff
@@ -76,6 +76,7 @@ export const submitDiagnosisKeys = async (
   cb: (errorMessage: string, successMessage: string) => void,
 ): Promise<void> => {
   keySubmissionModule.postDiagnosisKeys(cb);
+  permissionsModule.requestExposureNotificationAuthorization(cb);
```

Comment:
Why is this request necessary in this function?  It seems a bit out of place to me here. should we be requesting permissions earlier in the flow?

Reply:
Great catch. That was a mistake. Removing.